hooks.go

```diff
@@ -1,12 +1,14 @@
 package fiber
 
+import "os"
+
 // Handlers define a function to create hooks for Fiber.
 type OnRouteHandler = func(Route) error
 type OnNameHandler = OnRouteHandler
 type OnGroupHandler = func(Group) error
 type OnGroupNameHandler = OnGroupHandler
 type OnListenHandler = func() error
-type OnShutdownHandler = OnListenHandler
+type OnShutdownHandler = func(*os.Signal)
```

Comment:
This would break backwards compatibility :(. Honestly I think not showing `os.Signal` is fine for now. Are there any situations where a hook would need to change its functionality based on the signal type?

Reply:
I think we can add DisableDefaultGraceful 
config instead. Otherwise, i dont think hardcoded things are good. What about? @ReneWerner87 @hi019 @Streamer272

Reply:
Sure, works for me. What's hardcoded, though?

Reply:
> Sure, works for me. What's hardcoded, though?

Some users may want to add their custom signals.